server/src/routes/posts.js

```diff
@@ -52,4 +53,22 @@ router.delete('/:id', async (req, res) => {
   }
 });
 
+// Commenting once and updating post schema with comment id
+// eslint-disable-next-line no-unused-vars
+router.post('/:id/comment', async (req, res) => {
+  const comment = new Comment({
+    comment_id: req.body.comment_id,
+    body: req.body.body,
+  });
+  try {
+    const newComment = await comment.save();
+    res.status(201).json(newComment);
+
+    await Post.update({ _id: req.params.id }, { $push: { comment_id: newComment._id } });
+    res.status(200).send();
+  } catch (err) {
+    res.status(404).json({ message: err.message });
```

Comment:
Is 404 the right status code to send back in this case? I believe, it actually should be 400.